soroban-sdk/src/deploy.rs

```diff
@@ -12,38 +12,54 @@
 //! ### Examples
 //!
 //! ```
-//! # use soroban_sdk::{contract, contractimpl, BytesN, Env, Symbol};
+//! # use soroban_sdk::{contract, contractimpl, BytesN, Env, Symbol, IntoVal};
 //! #
+//! # const DEPLOYED_WASM: &[u8] = include_bytes!("../doctest_fixtures/contract.wasm");
+//! # const DEPLOYED_WASM_WITH_CTOR: &[u8] = include_bytes!("../doctest_fixtures/contract_with_constructor.wasm");
```

Comment:
Can you add a second example that uses the ctor, so that there are two examples each minimally focused on an single example topic? i.e. Each example has a single deploy function, one calls the ctor, one calls the non-ctor. Why: Examples are easier to digest when they do one thing.

Reply:
Done.